src/parameters/definition_types.ts

```diff
@@ -147,7 +147,7 @@ interface OAuth2ParameterDefinition extends BaseParameterDefinition<string> {
   /** Specifies the oauth2 provider this input is associated to */
   oauth2_provider_key: string;
   /** Dictates whether only the auth of the user running the workflow should be used */
```

Comment:
This defaults to `false`, yes? If so, let's clarify that:
```suggestion
  /** Dictates whether only the auth of the user running the workflow should be used. Defaults to `false`. */
```